index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<!-- Link CSS -->
+	<link rel="stylesheet" href="./styles/style.css">
+	<!-- Font Link -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap"
+		rel="stylesheet">
+	<title>Hernán Zamora Portfolio</title>
+</head>
+
+<body>
+
+	<!-- Nav Bar -->
+	<header>
+		<div class="nav-bar">
+			<a href="#">Hernán Zamora</a>
+			<!-- Nav -->
+			<nav class="menu">
+				<div class="bar"></div>
+				<div class="bar"></div>
+				<div class="bar"></div>
+			</nav>
+		</div>
+	</header>
```

Comment:
- [ ] **[OPTIONAL]:** You did great applying the change suggested by the previous reviewer. 💯 However, I think that the `<div class="nav-bar">` is not needed here. Instead, It would be a better idea to move the class to your `<header>` element and remove the `<div>`. This way you get the same design, and less markup. 🤓 🔧 


- [ ] **[OPTIONAL]:** Please also consider fixing the indentation, it improves a lot the readability of your code 🤓 

```suggestion
	<header class="nav-bar">		
	  <a href="#">Hernán Zamora</a>
	  <!-- Nav -->
	  <nav class="menu">
	    <div class="bar"></div>
	    <div class="bar"></div>
            <div class="bar"></div>
	  </nav>	
	</header>
```